### PR TITLE
Feature/wp compose tc rm

### PR DESCRIPTION
**Pull request by Rochius**
Hi team! here is my code with the integration test cases. Can you review it please?

### CODE REVIEW

File: wp_api/tests/integration/e2e_create_users_categories.robot [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+Resource    ../../keywords/categories/categories_crud.robot
+Resource    ../../keywords/categories/categories_crud_with_errors.robot
+Resource    wp_api/keywords/users/keywords.robot
+Resource    wp_api/keywords/pages/crud.robot
+Resource    wp_api/keywords/categories/common_keywords.robot
+Test Setup    Setup Test User and Categories
+Test Teardown    Teardown Testcase
+Force Tags    SMOKE    INTEGRATION    USERS    CATEGORIES    USERS_CATEGORIES    CREATE
+
+*** Keywords ***
```
Comment (TelmaRF) [inline wp_api/tests/integration/e2e_create_users_categories.robot]: Please remove line 15 my friend

**Review by TelmaRF — APPROVED** [review]
LGTM!

File: wp_api/tests/integration/e2e_create_users_categories.robot (hunk 1 above)
Comment (Rochius) [reply wp_api/tests/integration/e2e_create_users_categories.robot]: I already removed the line. Thank you!

**Review by Rogerrenjifo — APPROVED** [review]
LGTM.! 👍

**Review by DenisseC02 — APPROVED** [review]
Great, LGTM!

**Review by TelmaRF — APPROVED** [review]

**Review by daniskuvc — APPROVED** [review]
Thank you Ro

**Review by MartinJAA — COMMENTED** [review]
LGTM!

**Review by MartinJAA — APPROVED** [review]

**Review by Rogerrenjifo — APPROVED** [review]
Good Job! 👍

**Pull request merged**